h/templates/includes/annotation_card.html.jinja2

```diff
@@ -39,5 +39,12 @@
   <section class="annotation-card__text">
     {{ annotation.text_rendered|safe }}
   </section>
+  <footer class="annotation-card__footer">
+    {% if incontext_link %}
+      <a href="{{ incontext_link }}" target="_blank" title="Visit link in context">
```

Comment:
`"Visit link in context"` should be `"{% trans %}Visit link in context{% endtrans %}"` so that it gets internationalized. Some docs about this here http://docs.ckan.org/en/latest/contributing/string-i18n.html